### ISSUE TITLE
Default language should be better documented

### ISSUE THREAD

**Issue by lucmoco** [SharePoint/sp-dev-docs#1665]
"You can currently only create English sites." : this is not correct. We have three tenants and in 1 new communication sites are always created in English and in 2 others they are always created in German. It would be nice to know which parameter governs the default language and how to change it.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: f2d1ea72-8ceb-201c-e2a5-ce75033d246b
* Version Independent ID: 61532d1f-45ab-1100-0fc4-4db441b4c949
* Content: [Create SharePoint Communication site using REST](https://docs.microsoft.com/en-us/sharepoint/dev/apis/communication-site-creation-rest)
* Content Source: [docs/apis/communication-site-creation-rest.md](https://github.com/SharePoint/sp-dev-docs/blob/master/docs/apis/communication-site-creation-rest.md)
* Service: **unspecified**
* GitHub Login: @spdevdocs
* Microsoft Alias: **spdevdocs**

**Comment by advdberg** [comment]
We found out it's the default language of the root site (at https://<tenant>.sharepoint.com) which is leading to the default language of the created modern (team/communication) sites. Removing this (classic) root site and recreating it in the preferred language is currently the only way to influence it, which is pretty annoying...

**Comment by lucmoco** [comment]
Thanks @advdberg. This is useful information.

I also think we need a way to specify the default language of the site being created. This is a common scenario, especially in a country like Switzerland with several languages.

**Comment by advdberg** [comment]
Agree, I've seen that the language option is going to be added in the UI for creating modern sites [(https://techcommunity.microsoft.com/t5/Microsoft-SharePoint-Blog/Updates-to-SharePoint-self-service-site-creation/ba-p/218971)](url), hope it will then (or soon) also be working in the API's

**Comment by advdberg** [comment]
Just noticed it's fixed in version [3.3.1811.0](https://github.com/SharePoint/PnP-PowerShell/blob/082278cc6ac8407cf8535c5698ef1f97098ab4ea/CHANGELOG.md) of the SharePointPnPPowerShellOnline module

Did a quick test:
`New-PnPSite -Type TeamSite -Title "AVDBTest-NL-20181112a" -Alias "AVDBTest-NL-20181112a" -Lcid 1043`
Created a modern teamsite with Dutch default language on an 'English' tenant.
